rock_paper_scissors/rps.py

```diff
@@ -3,12 +3,17 @@
 import sys
 
 def rock_paper_scissors(n):
-  pass 
+  array = []
+  options = ['rock', 'paper', 'scissors']
+  for i in range(0, len(options)):
```

Comment:
for i in options will work too here. 
Seems like you will be using recursion for this one.